src/raygun.js

```diff
@@ -1015,6 +1017,7 @@ var raygunFactory = function(window, $, undefined) {
   // Make the actual CORS request.
   function makePostCorsRequest(url, data, _successCallback, _errorCallback) {
     var xhr = createCORSRequest('POST', url, data);
+    xhr.setRequestHeader('Content-Type', 'text/plain;charset=UTF-8');
```

Comment:
Only question is why text/plain instead of text/json but I don't think it actually matters for us.

Reply:
Good question, the default content-type used currently is `text/plain` (since previous it was falling back to browser defaults). We can/should update it to use `application/json` instead but we need the API to send the appropriate `Access-Control-Allow-Headers` first.

We can look at updating this in the future :)